encord/utilities/common.py

```diff
@@ -1 +1,12 @@
+import datetime
+from typing import Any
+
+import dateutil
+
 ENCORD_CONTACT_SUPPORT_EMAIL = "support@encord.com"
+
+
+def convert_str_date_to_datetime(data):
```

Comment:
I've created this function in `common.py` because it's used in at least 2 places